src/js/tasks/tasks.js

```diff
@@ -719,15 +719,38 @@ subtask("setPrices", "Update Lido ARM's swap prices")
     undefined,
     types.boolean
   )
+  .addOptionalParam(
+    "priceOffset",
+    "Offset the 1Inch/Curve price by this amount in basis points (above for buy, below for sell, reversed if negative). eg 100 = 1%",
+    0,
+    types.float
+  )
   .setAction(async (taskArgs) => {
     const signer = await getSigner();
 
     const armAddress = await parseDeployedAddress(
       `${taskArgs.arm.toUpperCase()}_ARM`
     );
-    const armContract = await ethers.getContractAt("AbstractARM", armAddress);
+    const arm = await ethers.getContractAt([
```

Comment:
I think you missed doing a Hardhat compile
`npx hardhat compile`

the following works for me locally after doing a HH compile
```js
const armAddress = await parseAddress("ORIGIN_ARM");
const arm = await ethers.getContractAt("OriginARM", armAddress);
```